dom.bs

```diff
@@ -4279,7 +4281,7 @@ empty string instead, and then do as described below, switching on the interface
 
 <dl class=switch>
  <dt>{{Attr}}
- <dd><p><a>Set an existing attribute value</a> with <a>this</a> and the given value.
+ <dd><p><a>Set an existing attribute value</a> with <a>this</a>, the given value.
```

Comment:
I don't understand this change.

Reply:
apologies some of this is because stuff was added and then removed and I've clearly not reverted stuff very well